protobuf/v1/snapshot.proto

```diff
@@ -0,0 +1,86 @@
+syntax = "proto3";
+
+import "google/protobuf/empty.proto";
+import "google/protobuf/wrappers.proto";
+import "google/protobuf/timestamp.proto";
+import "nexus.proto";
+package mayastor.v1;
+
+service SnapshotRpc {
+    // Common snapshot related methods.
+    rpc CreateReplicaSnapshot(CreateReplicaSnapshotRequest) returns (CreateReplicaSnapshotResponse) {}
+    rpc CreateNexusSnapshot(NexusCreateSnapshotRequest) returns (NexusCreateSnapshotResponse) {}
+    rpc ListSnapshot(ListSnapshotsRequest) returns (ListSnapshotsResponse) {}
+    rpc DeleteSnapshot(DeleteSnapshotRequest) returns (google.protobuf.Empty) {}
```

Comment:
nit: Should be Destroy rather than Delete as per the other services (eg: DestroyNexus/DestroyReplica)

Reply:
Accepted.